addons/web_editor/static/src/js/editor/odoo-editor/src/powerbox/Powerbox.js

```diff
@@ -325,6 +327,12 @@ export class Powerbox {
         if (ev.key === 'ArrowDown' || ev.key === 'ArrowUp') {
             ev.preventDefault();
         } else {
+            // Update initialTarget in case of backspace.
```

Comment:
Why is this needed, and why specifically in case of backspace? If it's because the node is changed when removing a character, wouldn't it be the case with delete as well?

Note to @Goaman because the question probably holds before this change already, but what if the user pressed "Home", "End", "Page Up", "Page Down" or "Enter"? I tried with "Page down" and the powerbox disappears but reappears as soon as my cursor reenters the node, which is kind of strange isn't it? Should we create a task for this?

Reply:
Yes it is because the node is changed when removing the character it would be also a case when we delete the last character. I have updated the comment.